app/views/groups/index.html.erb

```diff
@@ -0,0 +1,14 @@
+<p style="color: green"><%= notice %></p>
```

Comment:
Is `notice` your flash messages?
If so, this probably should be in you `application` layout.